### ISSUE TITLE
Leaflet isn't rendering even with basic command

### ISSUE THREAD

**Issue by sl4269** [rstudio/leaflet#688]
Even when I just us leaflet() the map doesn't show.  I am not sure why all of a sudden it is not working even though I've used it a lot. 
<img width="856" alt="Screen Shot 2020-06-15 at 10 00 38 AM" src="https://user-images.githubusercontent.com/54939714/84666443-128f0100-aeef-11ea-81ab-62901041625d.png">

**Comment by schloerke** [comment]
Related? https://github.com/rstudio/leaflet/issues/422

**Comment by tim-salabim** [comment]
@sl4269 what do you expect? `leaflet()` gives a blank map canvas just like in your screenshot:

``` r
library(leaflet)

leaflet()
```

![](https://i.imgur.com/bwGMkeP.png)

<sup>Created on 2020-06-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by jcheng5** [comment]
To add to what @tim-salabim said, if you need map tiles, call e.g. `leaflet() %>% addTiles()`

**Comment by sl4269** [comment]
I see but when even when I run this code it doesn't work.

`r
leaflet::leaflet(
  width = "100%",
          options = leafletOptions(zoomControl = FALSE)) %>% 
  leaflet::addProviderTiles("Stadia.AlidadeSmooth") `

**Comment by sl4269** [comment]
> Related? #422

Its hard to follow what the problem is exactly in that thread

**Comment by sl4269** [comment]
> @sl4269 what do you expect to happen? `leaflet()` gives a blank map canvas just like in your screenshot:
> 
> ```r
> library(leaflet)
> 
> leaflet()
> ```
> 
> ![](https://camo.githubusercontent.com/a6259fec88d06f1e7db56d0dec0e7c658a44bc92/68747470733a2f2f692e696d6775722e636f6d2f6277474d6b65502e706e67)
> 
> Created on 2020-06-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)

I forgot to add add_tiles() to my original code.

**Comment by sl4269** [comment]
Even when I use this code it isn't working.  I still get a gray screen.

leaflet() %>%
  addProviderTiles("providers$CartoDB.Positron") %>%  # Add default OpenStreetMap map tiles
  addMarkers(lng=174.768, lat=-36.852, popup="The birthplace of R")

**Comment by tim-salabim** [comment]
I think you need to remove the "" around providers$CartoDB.Positron

**Comment by sl4269** [comment]
> I think you need to remove the "" around providers$CartoDB.Positron

Thank you it works now!
